### PR TITLE
Initial support for Netberg 610, 710, 750 Aurora Switches

### PR DESCRIPTION
**Pull request by andrewsapronov**
Please merge initial support for barefoot based NetBerg Switches.

### CODE REVIEW

**Comment by pudelkoM** [comment]
Looks good to me.
Two things to fix:

- Sign a CLA as instructed
- Can you confirm that the directory names (`x86-64-netberg-aurora-610-r0`) are identical to the onl platform names under `/etc/onl/platform`? There is some program logic that depends on this fact.

**Comment by pudelkoM** [comment]
Since you did not allow edits by maintainers, you will also have to add these lines to the BUILD file yourself:

```Bazel
platform_config_test("x86-64-netberg-aurora-610-r0")
platform_config_test("x86-64-netberg-aurora-710-r0")
platform_config_test("x86-64-netberg-aurora-750-r0")
```

**Comment by andrewsapronov** [comment]
Hi Maximilian, 

CLA has been signed.

I've added our platforms to the BUILD and we confirm that `/etc/onl/platform` is identical to directory names.

Could you please also advise what kind of permission you mentioned?

--
BR, 
Andrew

**Comment by pudelkoM** [comment]
> Could you please also advise what kind of permission you mentioned?

GH has the option to allow edits by maintainers (us): https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/allowing-changes-to-a-pull-request-branch-created-from-a-fork
It's not needed anymore since you did the changes yourself.

Thanks for checking the names. I think we can merge this now.

**Review by pudelkoM — DISMISSED** [review]

File: stratum/hal/config/x86-64-netberg-aurora-610-r0/port_map.json
Comment (Yi-Tseng) [inline stratum/hal/config/x86-64-netberg-aurora-610-r0/port_map.json:2]: `support on x86-64-netberg-aurora-610-r0` ?

File: stratum/hal/config/x86-64-netberg-aurora-710-r0/port_map.json
Comment (Yi-Tseng) [inline stratum/hal/config/x86-64-netberg-aurora-710-r0/port_map.json:2]: `x86-64-netberg-aurora-710-r0` ?

Comment (Yi-Tseng) [reply stratum/hal/config/x86-64-netberg-aurora-710-r0/port_map.json:2]: Maybe this is a wrong port map file for Aurora 710 since it has more than 32 ports(connectors).
Can you check again?

File: stratum/hal/config/x86-64-netberg-aurora-750-r0/port_map.json
Comment (Yi-Tseng) [inline stratum/hal/config/x86-64-netberg-aurora-750-r0/port_map.json:2]: I think this should be the correct file for Aurora 710.

**Review by Yi-Tseng — CHANGES_REQUESTED** [review]

**Comment by pudelkoM** [comment]
Looks good.
I'll leave it to Yi to merge this

**Review by Yi-Tseng — APPROVED** [review]

**Comment by pudelkoM** [comment]
@andrewsapronov please rebase this PR on latest master, otherwise we can't merge. Or enable maintainer edits as explained above.

**Comment by andrewsapronov** [comment]
> @andrewsapronov please rebase this PR on latest master, otherwise we can't merge. Or enable maintainer edits as explained above.

@pudelkoM  I've made rebase. Please check. 

BTW, I spent some time but I couldn't find 'tick' for enabling edits :(

**Comment by pudelkoM** [comment]
Thanks!

Here is the checkbox:
Capture

**Review by pudelkoM — APPROVED** [review]

**Pull request merged**
